### PR TITLE
Passive A Assassin, Slash fix

### PR DESCRIPTION
**Pull request by Polisvh**
## Describe your changes
Recall and SmokeBomb rescalling
Change both to level 3 skills with proper values

Slash yellow numbers changed back to static values
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested my changes.

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/sword/Slash.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ public void loadSkillConfig() {
         damageIncreasePerLevel = getConfig("damageIncreasePerLevel", 1.5, Double.class);
         distance = getConfig("distance", 5.0, Double.class);
         distanceIncreasePerLevel = getConfig("distanceIncreasePerLevel", 0.0, Double.class);
-        cooldownReduction = getConfig("cooldownReduction", 3.0, Double.class);
+        cooldownReduction = getConfig("cooldownReduction", 4.0, Double.class);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/sword/Slash.java:164]: i dont think you need to buff slash

**Review by Mykindos — CHANGES_REQUESTED** [review]

**Comment by Mykindos** [comment]
Rather than changing the base durations for these skills, I'd rather buff the increase per level

**Comment by Polisvh** [comment]
> Rather than changing the base durations for these skills, I'd rather buff the increase per level

In recall's case that would make more sense to invest more skill points for better duration, tho in SmokeBomb I believe the cooldown is the bigger reward for investing more skill points - most people dont sit invisible that long anyway
What are your suggestions for recall (I just used mineplex scaling I believe)

**Comment by Mykindos** [comment]
> > Rather than changing the base durations for these skills, I'd rather buff the increase per level
> 
> In recall's case that would make more sense to invest more skill points for better duration, tho in SmokeBomb I believe the cooldown is the bigger reward for investing more skill points - most people dont sit invisible that long anyway What are your suggestions for recall (I just used mineplex scaling I believe)

Recall, base duration 2.5, extra 0.5 seconds per level
Effectively 3.5 seconds at level 3

We dont want to buff assassin too much

**Comment by Polisvh** [comment]
> > > Rather than changing the base durations for these skills, I'd rather buff the increase per level
> > 
> > 
> > In recall's case that would make more sense to invest more skill points for better duration, tho in SmokeBomb I believe the cooldown is the bigger reward for investing more skill points - most people dont sit invisible that long anyway What are your suggestions for recall (I just used mineplex scaling I believe)
> 
> Recall, base duration 2.5, extra 0.5 seconds per level Effectively 3.5 seconds at level 3
> 
> We dont want to buff assassin too much

Dude that is worse than it is right now tho lmao

**Comment by Mykindos** [comment]
> > > > Rather than changing the base durations for these skills, I'd rather buff the increase per level
> > > 
> > > 
> > > In recall's case that would make more sense to invest more skill points for better duration, tho in SmokeBomb I believe the cooldown is the bigger reward for investing more skill points - most people dont sit invisible that long anyway What are your suggestions for recall (I just used mineplex scaling I believe)
> > 
> > 
> > Recall, base duration 2.5, extra 0.5 seconds per level Effectively 3.5 seconds at level 3
> > We dont want to buff assassin too much
> 
> Dude that is worse than it is right now tho lmao

oh right, i thought it had no scaling on duration

then start at 2.0 and increase by 1.5 per level, this is 5 seconds at level 3

**Review by Mykindos — APPROVED** [review]

**Pull request merged**